src/components/issues/Card.module.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+	padding: 1rem;
+	width: 100%;
+	background: #fff;
+	border: 1px solid #d9d9d9;
+	border-radius: 0.5rem;
+	transition: 0.2s ease-out;
+	display: flex;
+	flex-direction: column;
+	margin: 2rem auto;
+	justify-content: space-between;
+	position: relative;
+}
+
+.card:hover {
+	border-color: #e5e5e5;
+	filter: drop-shadow(0 20px 13px rgba(0, 0, 0, 0.02))
+		drop-shadow(0 8px 5px rgba(0, 0, 0, 0.06));
+}
```

Comment:
can nest this in the above block as `&:hover`

Reply:
Done